OpenCV 4.5.1 cannot properly initialize VTK 9.0.1 during build with opencv_world

Sir，
I compiled opencv4.5.1 with a line surface error

```
CMake Error at C:/Program Files (x86)/VTK/lib/cmake/vtk-9.0/vtkModule.cmake:2955 (get_property):
  get_property could not find TARGET opencv_viz.  Perhaps it has not yet been
  created.
Call Stack (most recent call first):
  E:/opencv_contrib-4.5.1/modules/viz/CMakeLists.txt:43 (vtk_module_autoinit)
  modules/world/CMakeLists.txt:13 (include)
  modules/world/CMakeLists.txt:32 (include_one_module)


CMake Error at C:/Program Files (x86)/VTK/lib/cmake/vtk-9.0/vtkModule.cmake:2962 (target_compile_definitions):
  Cannot specify compile definitions for target "opencv_viz" which is not
  built by this project.
Call Stack (most recent call first):
  E:/opencv_contrib-4.5.1/modules/viz/CMakeLists.txt:43 (vtk_module_autoinit)
  modules/world/CMakeLists.txt:13 (include)
  modules/world/CMakeLists.txt:32 (include_one_module)
```
But I've compiled vtk9.0.1.

Reproduced with Ubuntu 18.04 and VTK 9.0.1. CMake command line:
```
$ cmake -DBUILD_opencv_world=ON -DWITH_VTK=ON -DOPENCV_EXTRA_MODULES_PATH=../opencv_contrib/modules ../opencv-master
```
CMake output:
```
-- Allocator metrics storage type: 'long long'
--     module opencv_flann...
--     module opencv_imgproc...
--     module opencv_intensity_transform...
--     module opencv_ml...
--     module opencv_phase_unwrapping...
--     module opencv_photo...
--     module opencv_plot...
--     module opencv_quality...
--     module opencv_reg...
--     module opencv_surface_matching...
--     module opencv_viz...
CMake Error at /usr/local/lib/cmake/vtk-9.0/vtkModule.cmake:2955 (get_property):
  get_property could not find TARGET opencv_viz.  Perhaps it has not yet been
  created.
Call Stack (most recent call first):
  /home/alexander/Projects/OpenCV/opencv_contrib/modules/viz/CMakeLists.txt:43 (vtk_module_autoinit)
  modules/world/CMakeLists.txt:13 (include)
  modules/world/CMakeLists.txt:32 (include_one_module)


CMake Error at /usr/local/lib/cmake/vtk-9.0/vtkModule.cmake:2962 (target_compile_definitions):
  Cannot specify compile definitions for target "opencv_viz" which is not
  built by this project.
Call Stack (most recent call first):
  /home/alexander/Projects/OpenCV/opencv_contrib/modules/viz/CMakeLists.txt:43 (vtk_module_autoinit)
  modules/world/CMakeLists.txt:13 (include)
  modules/world/CMakeLists.txt:32 (include_one_module)


--     module opencv_xphoto...
--     module opencv_alphamat...
```

--- a/modules/world/CMakeLists.txt
+++ b/modules/world/CMakeLists.txt
@@ -59,6 +59,10 @@ ocv_module_include_directories()
 #message(STATUS "${OPENCV_MODULE_${the_module}_SOURCES}")
 ocv_create_module(${link_deps})
 
+if(";${OPENCV_MODULES_BUILD};" MATCHES ";opencv_viz;" AND OPENCV_MODULE_opencv_viz_IS_PART_OF_WORLD AND VTK_VERSION VERSION_GREATER_EQUAL "8.90.0")
+  vtk_module_autoinit(TARGETS opencv_world MODULES ${VTK_LIBRARIES})
+endif()
+
 ocv_target_compile_definitions(${the_module} PRIVATE OPENCV_MODULE_IS_PART_OF_WORLD=1)
 
 if(BUILD_opencv_imgcodecs AND OPENCV_MODULE_opencv_imgcodecs_IS_PART_OF_WORLD)

